libs/hpc-ui/src/lib/components/form-fields/autocomplete-field.tsx

```diff
@@ -0,0 +1,66 @@
+import { Autocomplete, AutocompleteProps } from '@mui/material';
+import { useField, useFormikContext } from 'formik';
+import tw from 'twin.macro';
+import { StyledTextField } from './text-field';
+import { FormObjectValue } from '@unocha/hpc-data';
+
+const StyledAutocomplete = tw(Autocomplete)`
+  min-w-[10rem]
+  w-full
+`;
+
+const AutocompleteSelect = ({
+  name,
+  label,
+  options,
+  placeholder,
+  readOnly,
+  isMulti,
+  ...otherProps
+}: {
+  name: string;
+  label: string;
+  options: Array<FormObjectValue>;
+  placeholder?: string;
+  readOnly?: boolean;
+  isMulti?: boolean;
+}) => {
```

Comment:
For this comment, in the case of `AutocompleteSelect` There are some props that are not used in this branch, and in the rest of my work I didn't use it either, so I removed them from the component 👍 , I thought that even if I didn't use them it couldn't hurt to let it implemented since it was easy to do, but I understand that when in the future we need (for example) `multiple` AutocompleteSelect we should build it for it to satisfy the problem we face at the moment.

For the rest of inputs, there some unused props, but in my work on top of this branch I end up using them, so i didn't remove them.

For this kind of PR I think we should be a bit more flexible since is not finished work, once I finish I will review myself everything and in that moment I think is nice if you are very picky so everything can be addressed because it has to be.